spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -3,18 +3,93 @@
 RSpec.describe Api::V1::KeywordsController, type: :request do
   describe 'GET#index' do
     it 'returns success status' do
-      get api_v1_keywords_path
+      header, = create_token_header
```

Comment:
just small suggestion, 🎨  we can define a variable with `_` underscore to mark it as unused, and it makes the pattern matching become clearer.

```suggestion
      header, _user = create_token_header
```

Reply:
I updated `create_token_header` to return only header 🙏